bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -45,34 +73,42 @@ else:
             newsnr = pycbc.events.ranking.newsnr(snr, chisq)
             
             found_names += ["%s SNR" % ifo, "CHISQ", "NewSNR"]
-            found_cols += [snr, chisq, newsnr]
+            found_cols += [numpy.round(snr,2), numpy.round(chisq,2), numpy.round(newsnr,2)]
             found_formats += ['##.##', '##.##', '##.##']
-          
-dec_dist = numpy.maximum(inj['eff_dist_h'][:][idx], inj['eff_dist_l'][:][idx])
+
+eff_dist = {'eff_dist_l':'Eff Dist(L)','eff_dist_h':'Eff Dist(H)','eff_dist_v':'Eff Dist(V)',
+            'eff_dist_k':'Eff Dist(K)','eff_dist_i':'Eff Dist(I)'}
+keys = inj.keys()
+eff_dist_str = []
+eff_distance = []
+eff_dist_format = []
+for dist in eff_dist :
+    if(dist in keys):
+        eff_distance.append(numpy.round(inj[dist][:][idx],2))
+        eff_dist_str.append(eff_dist[dist])
+        eff_dist_format.append('##.##')
+                
+dec_dist = numpy.max(eff_distance,0)
 m1, m2 = inj['mass1'][:][idx], inj['mass2'][:][idx]
 mchirp, eta = pycbc.pnutils.mass1_mass2_to_mchirp_eta(m1, m2)
 dec_chirp_dist = pycbc.pnutils.chirp_distance(dec_dist, mchirp)
 
-columns = [dec_chirp_dist, inj['end_time'][:][idx], m1, m2, mchirp, eta,
-           inj['spin1x'][:][idx], inj['spin1y'][:][idx], inj['spin1z'][:][idx],
-           inj['spin2x'][:][idx], inj['spin2y'][:][idx], inj['spin2z'][:][idx],
-           inj['distance'][:][idx],
-           inj['eff_dist_h'][:][idx], inj['eff_dist_l'][:][idx], 
-           ] + found_cols
+columns = [numpy.round(dec_chirp_dist,2), numpy.round(inj['end_time'][:][idx],2), 
+           numpy.round(m1,2), numpy.round(m2,2), numpy.round(mchirp,2), numpy.round(eta,2),
+           numpy.round(inj['spin1x'][:][idx],2), numpy.round(inj['spin1y'][:][idx],2), numpy.round(inj['spin1z'][:][idx],2),
+           numpy.round(inj['spin2x'][:][idx],2), numpy.round(inj['spin2y'][:][idx],2), numpy.round(inj['spin2z'][:][idx],2),
+           numpy.round(inj['distance'][:][idx],2)] + eff_distance + found_cols
```

Comment:
is the rounding not done in the html_table already?

Reply:
When you write 'N/A' as one of the entries then by default your matrix no longer remain a list of float but converts into a list of strings. The '##.##' can't deal with it. Checked.

Reply:
Ah thank you, makes sense